dali/test/python/test_operator_random_bbox_crop.py

```diff
@@ -208,10 +210,12 @@ def check_crop_dims_variable_size(anchor, shape, scaling, aspect_ratio):
 
 def check_crop_dims_fixed_size(anchor, shape, expected_crop_shape, input_shape):
     ndim = len(shape)
+    assert(anchor.dtype == np.int32)
+    assert(shape.dtype == np.int32)
     for d in range(ndim):
         assert(anchor[d] >= 0.0 and anchor[d] <= input_shape[d])
         assert shape[d] == expected_crop_shape[d], "{} != {}".format(shape, expected_crop_shape)
-        assert(anchor[d] + shape[d] > 0.0 and anchor[d] + shape[d] <= input_shape[d])
+        assert(anchor[d] + shape[d] > 0.0)
```

Comment:
why removing the second check?

Reply:
Bottom right corner can be outside original image boundaries if crop_shape > input_shape.